### DISCUSSION TITLE
Extracellular field visualization

### DISCUSSION THREAD

**Discussion by PALMS-gui** [wmglab-duke/ascent#20] [Q&A] [answered]
Is that somehow possible to visualize extracellular electric fields? Generated input files contain only values corresponding to fibers' location, but not the whole 3D space.

**Comment by thedannymarsh** [comment, marked as answer]
Hi PALMS-gui,

Generally to visualize the electric fields across the model, we use COMSOL's plotting features. Opening up the bases and doing something like a multislice 3D plot, where the variable of interest is "V." If you are curious about how bases function you can read more under "active_srcs" in the documentation here: https://wmglab-duke-ascent.readthedocs.io/en/latest/S8-JSON-file-parameter-guide.html?highlight=bases.

Hope this helps!
